### ISSUE TITLE
Debugging not working

### ISSUE THREAD

**Issue by Kesymaru** [willyelm/xatom-debug#26]
Hello.

I have installed and configured to debug my project on node.js, the configuration is:

Binary Path: "C:\Program Files\nodejs\node.exe"
Port: 8000
Source Script: C:\Users\myUser\Documents\Project\server\bin\project
Environment Variables:  NODE_ENV=development

The problem is that the server runs correctly, when the project is running correctly the breakpoints are not working, adding a new breakpoint and tested sending a request to the server and the breakpoint is not fired for some reason.

The xatom-debug shows a status "Node.js: Debugger Resumed"

On the node console the output is:
```
Debugger attached.
Debugger listening on port 5865.
Warning: This is an experimental feature and could change at any time.
To start debugging, open the following URL in Chrome:
    chrome-devtools://devtools/remote/serve_file/@60cd6e859b9f557d2312f5bf532f6aec5f284980/inspector.html?experiments=true&v8only=true&ws=localhost:5865/fc0912a5-ffca-43d4-a90b-d12e0fba2390
Debugger attached.
```
Also tried to debug the project using the chrome inspector and the same problem happened.

I would like to know if i am missing something on my setup or if there is another way to debug a project (node + express + restful api), i have been using webstorm node debugger for the same project with the same setup and it works on webstorm.

**Comment by dhruvr** [comment]
@Kesymaru  It Happened with me, because already a node server was running and executing my script in the background. Close your debugger and check if server is already running?

**Comment by Kesymaru** [comment]
@dhruvr i tested closing my debugger and run it again, even killing the debugger process from the task manager. the server runs correctly after killed it and run it again, but the debugger does not work.

**Comment by dhruvr** [comment]
@Kesymaru  Are you starting the debugger and server both. The debugger is supposed to start your node process?

**Comment by Kesymaru** [comment]
@dhruvr yeah the debugger runs the main script of my project /server/bin/project so basically when i run the debugger it starts my server project

**Comment by Kesymaru** [comment]
Does anyone know how to use correctly the xatom debugger?

I want to start the app and the debugger, the idea is that when i run the debugger it start my app in debug mode.

It is posible to attach the debugger to an already started app?

**Comment by Kesymaru** [comment]
Does anyone know how to make the debugger work, starting the app and then debugging it?

It is possible to attach the debugger to an already stated app?

I really want to make the debugger work so i can use atom as my daily editor.

**Comment by willyelm** [comment]
Hi @Kesymaru, are you using a nodejs right away or you have it compiled with a custom sourcemaps directory?
